src/System.Management.Automation/engine/InvocationInfo.cs

```diff
@@ -451,7 +451,7 @@ internal void ToPSObjectForRemoting(PSObject psObject)
     /// <summary>
     /// A CommandInfo that has been serialized/deserialized as part of an InvocationInfo during a remote invocation.
     /// </summary>
-    public class RemoteCommandInfo : CommandInfo
+    public sealed class RemoteCommandInfo : CommandInfo
```

Comment:
Causes test failure due to public API change:

https://dev.azure.com/powershell/PowerShell/_build/results?buildId=45285&view=ms.vss-test-web.build-test-results-tab&runId=1310450&resultId=100640&paneView=debug

Reply:
reverted